<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMainWindow</class>
 <widget class="QMainWindow" name="MyMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,1">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>1200</width>
            <height>700</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::WinPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="text">
           <string>Main image goes here</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;H1&amp;gt;Welcome to Lac's Image Processor!&amp;lt;/H1&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,2,1,1">
        <item>
         <layout class="QHBoxLayout" name="detectMethodLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Detect Method</string>
            </property>
            <property name="buddy">
             <cstring>detectSelectionBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="detectSelectionBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="detectThresholdLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Threshold</string>
            </property>
            <property name="buddy">
             <cstring>detectThresholdBox</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="detectThresholdBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="startDetectButton">
          <property name="text">
           <string>Detect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="endDetectButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>466</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Start(&amp;S)</string>
    </property>
    <addaction name="actionLoad_Picture_L"/>
    <addaction name="actionSave_Picture_S"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuSimple_Processes_P">
    <property name="title">
     <string>Simple Processes(&amp;P)</string>
    </property>
    <addaction name="actionClear_C"/>
    <addaction name="actionCancel_C"/>
    <addaction name="actionNext_N"/>
    <addaction name="actionGray_Scale_G"/>
    <addaction name="actionRestore_R"/>
    <addaction name="actionDetect"/>
   </widget>
   <widget class="QMenu" name="menuMatching_M">
    <property name="title">
     <string>Matching(&amp;M)</string>
    </property>
    <addaction name="actionLoad_Matching_Picture_LMP"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuSimple_Processes_P"/>
   <addaction name="menuMatching_M"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad_Picture_L"/>
   <addaction name="actionSave_Picture_S"/>
   <addaction name="actionDetect"/>
   <addaction name="actionCancel_C"/>
   <addaction name="actionNext_N"/>
   <addaction name="actionGray_Scale_G"/>
   <addaction name="actionRestore_R"/>
   <addaction name="actionClear_C"/>
   <addaction name="actionExit_E"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Picture_L">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/insert-image.png</normaloff>:/image/image/insert-image.png</iconset>
   </property>
   <property name="text">
    <string>Load Picture(&amp;L)</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/exit.png</normaloff>:/image/image/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
  </action>
  <action name="actionGray_Scale_G">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/system-lock-screen.png</normaloff>:/image/image/system-lock-screen.png</iconset>
   </property>
   <property name="text">
    <string>Gray Scale(&amp;G)</string>
   </property>
  </action>
  <action name="actionRestore_R">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/edit-undo.png</normaloff>:/image/image/edit-undo.png</iconset>
   </property>
   <property name="text">
    <string>Restore(&amp;R)</string>
   </property>
  </action>
  <action name="actionMatch_M">
   <property name="text">
    <string>Match(&amp;M)</string>
   </property>
  </action>
  <action name="actionDetect">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/gtk-about.png</normaloff>:/image/image/gtk-about.png</iconset>
   </property>
   <property name="text">
    <string>Detect(&amp;D)</string>
   </property>
  </action>
  <action name="actionLoad_Matching_Picture_LMP">
   <property name="text">
    <string>Load Matching Picture(&amp;LMP)</string>
   </property>
  </action>
  <action name="actionCancel_C">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/back.png</normaloff>:/image/image/back.png</iconset>
   </property>
   <property name="text">
    <string>Last(&amp;L)</string>
   </property>
  </action>
  <action name="actionNext_N">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/forward.png</normaloff>:/image/image/forward.png</iconset>
   </property>
   <property name="text">
    <string>Next(&amp;N)</string>
   </property>
  </action>
  <action name="actionClear_C">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/edit-delete.png</normaloff>:/image/image/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Clear(&amp;C)</string>
   </property>
  </action>
  <action name="actionSave_Picture_S">
   <property name="icon">
    <iconset>
     <normaloff>:/image/image/filesave.png</normaloff>:/image/image/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save Picture(&amp;S)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
